docs/orchestrating-elastic-stack-applications/agent.asciidoc

```diff
@@ -0,0 +1,403 @@
+:page_id: elastic-agent
+:agent_recipes: https://raw.githubusercontent.com/elastic/cloud-on-k8s/{eck_release_branch}/config/recipes/elastic-agent
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Run Elastic Agent on ECK
+
+experimental[]
```

Comment:
I guess this is a message that we reuse and it is accurate, but "may be ... removed completely" feels discouraging :) Just an observation, nothing we need to fix.